Packages/com.unity.inputsystem/InputSystem/Editor/UITKAssetEditor/Commands/Commands.cs

```diff
@@ -149,6 +149,17 @@ public static Command PasteActionMaps()
             };
         }
 
+        public static Command PasteActionIntoActionMap(int actionMapIndex)
```

Comment:
I was suprised there is no destination index as argument to this function. Where do we handle a case such as copying an action inside one map, then switching to another map, selecting an existing action in the middle of that map and then pasting? I would personally expect the action to be pasted below the selected one in that case? Or maybe I am outside what was previously supported with this...

Reply:
During dragging an Action the visualization does not switch to show the ActionTreeView of the Action Map drop target. So Drag&Drop of an Action is only possible into an ActionMap, not into the TreeView of that Action Map. This is consistent to the behavior of the IMGUI editor. All actions get appended to the end of the action list of the drop target.

Reply:
I see, I think that's OK for now